pkgs/development/python-modules/OWSLib/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dateutil
+, pep8
+, pillow
+, pyproj
+, pytz
+, pytest
+, pytestcov
+, requests
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "OWSLib";
+  version = "0.16.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m05225g1sqd2i8r2riaan33953hfni9wjq7n225snhl7klsb5gc";
+  };
+
+  buildInputs = [ dateutil pep8 pillow pyproj pytz requests tox ];
```

Comment:
`cehckInputs`?

Reply:
Already resolved. Not sure why this diff is showing up for you.